app/validators/(2-guides)/mainnet/validator-setup/page.mdx

```diff
@@ -75,22 +71,21 @@ This machine must be available 24/7 and thus it is recommended that you only try
 
 You should ensure that all connections on ports `30333 (TCP)` and `8078 (TCP)` are not blocked! Moreover make sure your firewall allows NTP outbound traffic on port `123 (UDP)`
 
-Some VPS providers enable an all-port firewall by default, but if this is not the case your node will not be able to participate in Keygen / Signing ceremonies and as a result will never become active.
+Some VPS providers enable an all-port open firewall by default. We recommend opening only the required ports, but if this is not the case your node will not be able to participate in Keygen / Signing ceremonies and as a result will never become active.
```

Comment:
```suggestion
Some VPS providers enable an all-port open firewall by default, but if this is not the case your node will not be able to participate in Keygen / Signing ceremonies and as a result will never become active. We recommend opening only the required ports.
```

Sounds like previously the recommendation is to only open the required orts "but if thats not the case it wont work"